source/includes/_daily_campaign_activity.md

```diff
@@ -0,0 +1,278 @@
+### **Daily Campaign Acticvity**
+
+### Resource Overview
+
+|Method|URI Format|
+|---|---|
+|GET|/client_reports/daily_campaign_activity/[gmaid]?[query_params]|
+
+Use GET to retrieve information for the Daily Campaign Activity report for a given advertiser.
+
+The data returned will include impressions, leads, clicks, calls, qualified_web_events, web_events, emails, and spend values organized by campaign and campaign_cycle.
+
+### Parameters
+
+When using the GET method, the results can be filtered using these parameters:
+
+|Param|Function|
+|---|---|
+|start_date|Restricts the results to those occurring on or after this date|
+|end_date|Restricts the results to those occurring on or before this date|
+|global_master_campaign_id[]|Restrict results to one or more specific campaigns. This should be a comma separated string. Ex: global_master_campaign_id[]=TEST_1,TEST_2|
+|campaign_status[]|Restrict results to all campaigns with given status values.  Allowed values are running, stopped and ended. This should be a comma separated string. Ex: campaign_status[]=running,stopped|
+|campaign_cycle|Restrict results to a single campaign cycle|
+|interval_size|Use calendar_month or calendar_week to roll up the data points into calendar intervals (default is 1 day per interval)|
+
+
+To specify a date range:
+
+   - Specify start_date and end_date.
+   - When specifying a campaign_cycle, do not use date range params.
+
+### Response Data Details
+
+> Retrieve data for a specific range of dates
+
+```
+curl -H "Authorization: Bearer Bearer OAUTH_ACCESS_TOKEN \
+https://api.localiqservices.com/client_reports/daily_campaign_activity/TEST_1?start_date=2016-12-01&end_date=2016-12-31"
+```
+
+>  Retrieve data for a specific campaign starting on a certain date
+
+```
+curl -g -H "Authorization: Bearer Bearer OAUTH_ACCESS_TOKEN \
```

Comment:
This looks like a typo
```
Authorization: Bearer Bearer OAUTH_ACCESS_TOKEN \
```
Most likely this was copied from somewhere else, so I suspect it's incorrect there also.